lib/getLocationForIpWithService.js

```diff
@@ -0,0 +1,31 @@
+import { Meteor } from 'meteor/meteor';
+import Geoip2ws from 'geoip2ws';
+
+import { sanitizeServiceResult } from './sanitizeResult';
+
+const userId = Meteor.settings.maxmind.userId;
+const licenseKey = Meteor.settings.maxmind.licenseKey;
+
+const lookupIpWithService = new Geoip2ws({
+    userId,
+    licenseKey,
+    service: 'country',
+    requestTimeout: 2000,
+});
+
+export default function getLocationForIpWithService(ip) {
+    return new Promise((resolve, reject) => {
+        if (!userId || !licenseKey) {
```

Comment:
confused: thought this is the meteor userId - maybe we can name it maxmindUserId or something